client/lib/js/react/components/accounts.js

```diff
@@ -78,9 +80,27 @@ const AccountRow = React.createClass({
         activeAccount: React.PropTypes.object,
         handleClick: React.PropTypes.func
     },
-
+    
+    connectToLedger() {
+        const { account } = this.props;
+        return new BigchainDBLedger({
+            auth: {
+                account: {
+                    id: account.vk,
+                    uri: 'ws://localhost:8888/users/' + account.vk
+                }
+            },
+        });
+    },
+    
     handleClick() {
-        this.props.handleClick(this.props.account);
+        const { account, handleClick } = this.props;
+        const ledger = this.connectToLedger(account);
+        ledger.connect().catch((err) => {
```

Comment:
If this error is unrecoverable, display it to the user (somewhere where they can see it :D)